package.json

```diff
@@ -57,6 +60,7 @@
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.29.1",
+    "eslint-plugin-react-hooks": "^4.6.2",
```

Comment:
This package has been added to be able to use the rule 'react-hooks/exhaustive-deps'